packages/experimental/use-slots/package.json

```diff
@@ -33,7 +33,9 @@
   "dependencies": {
     "@fluentui-react-native/memo-cache": ">=0.2.10 <1.0.0",
     "@fluentui-react-native/use-styling": ">=0.2.1 <1.0.0",
-    "@uifabricshared/immutable-merge": ">=0.5.67 <1.0.0"
+    "@uifabricshared/foundation-settings": ">=0.6.1 <1.0.0",
+    "@uifabricshared/immutable-merge": ">=0.5.67 <1.0.0",
+    "react": "16.11.0"
```

Comment:
This should be a peerDep.  And should match the other peerDep range for react (since this dep will break people using RN 0.60)  (Same for other additions of react as a dep)

Reply:
Yes. Moved these into peerDeps and set to `^16.8.0`.